tests/Integration/SessionMigrationIntegrationTest.php

```
@@ -0,0 +1,280 @@
+        // Close session
+        session_write_close();
+
+        // Verify old session was deleted
+        self::assertFalse($connection->exists($oldSessionId), 'Old session should be deleted');
+
+        // Verify new session exists
+        self::assertTrue($connection->exists($newSessionId), 'New session should exist');
+
+        // Clean up
+        $connection->delete($newSessionId);
+    }
```

PHPMD (2.15.0)

testMigrateSuccessfully accesses the super-global variable $_SESSION. (undefined)

(Superglobals)